src/calc.c

```diff
@@ -64,6 +64,12 @@ typedef struct
 #define CALC_COMMAND_OPTION "-calc-command"
 
 
+/**
+ * Flag to disable bold results
+ */
+#define RESULT_BOLD_OPTION "-no-bold"
```

Comment:
It might make more sense to call this `NO_BOLD_OPTION` to be consistent and true to its effect.

Reply:
You're right, your suggestion makes more sense.